### ISSUE TITLE
Issue testing file content with current API

### ISSUE THREAD

**Issue by ianwalter** [yeoman/yeoman-assert#23]
Yo Yo...

I'm trying to test file content (multi-line strings) with `assert.fileContent` but the experience leaves a lot to be desired. I would like to compare the file content to strings using `assert.textEqual(body, regex)` instead of `body.indexOf(regex) !== -1` so that I can see a nicely formatted diff when the file content and passed string are not equal instead of the current, hard to parse, message: `${file} did not match '${regex}'. Contained:\n\n${body}` (if you need screenshots, I can take some). 

This presents a problem though because if I refactored `assert.fileContent` like I want above it would change the functionality so that it would have to be an exact match which obviously you don't want. So I propose adding a new assertion to handle this case. Is this change acceptable? I can submit a PR if someone can give me an assurance that it has a chance of getting accepted.

Thanks!

**Comment by SBoudrias** [comment]
Hry @ianwalter, I think that's something we can add. I'll make sure if you send a PR we'll follow up.
